### PR TITLE
Minor dark mode fixes

### PR DESCRIPTION
**Pull request by differsthecat**
## Type of change
- [x] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other

## Code changes

* **src/scss/variables.scss** Fix issues outlined here: https://app.asana.com/0/1153292148278596/1201274604201196/f

## Before you submit
- [x] I have checked for **linting** errors (`npm run lint`) (required)
- [ ] This change requires a **documentation update** (notify the documentation team)
- [ ] This change has particular **deployment requirements** (notify the DevOps team)

### CODE REVIEW

File: src/scss/variables.scss [hunk 1]
```diff
@@ -246,8 +246,8 @@ $themes: (
         badgePrimaryBackground: $darkLinks,
         badgePrimaryBackgroundHover: $darkPrimary-alt,
         badgePrimaryText: $darkDarkBlue2,
-        badgeSecondaryBackground: $darkDarkBlue2,
-        badgeSecondaryText: $white,
+        badgeSecondaryBackground: $secondary,
+        badgeSecondaryText: #212529,
```
Comment (eliykat) [inline src/scss/variables.scss:250]: These values for the badge background and text are different to what Danielle requested [here](https://app.asana.com/0/0/1201295377550891/1201307132191437/f), is that intentional?

Comment (differsthecat) [reply src/scss/variables.scss:250]: Ah I missed that update, I'll get it fixed!

Comment (differsthecat) [reply src/scss/variables.scss]: @eliykat Fixed!

Screen Shot 2021-11-03 at 9 26 52 PM

**Review by eliykat — APPROVED** [review]
Looks good!

**Pull request merged**